testnets/desmostestnet/chain.json

```diff
@@ -0,0 +1,119 @@
+{
+  "$schema": "../../chain.schema.json",
+  "chain_name": "desmostestnet",
+  "status": "live",
+  "network_type": "testnet",
+  "pretty_name": "Desmos Testnet",
+  "chain_id": "morpheus-apollo-3",
+  "bech32_prefix": "desmos",
+  "daemon_name": "desmos",
+  "node_home": "$HOME/.desmos",
+  "key_algos": [
+    "secp256k1"
+  ],
+  "slip44": 852,
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "udaric",
+        "fixed_min_gas_price": 0.001,
+        "low_gas_price": 0.01,
+        "average_gas_price": 0.03,
+        "high_gas_price": 0.05
+      }
+    ]
+  },
+  "staking": {
+    "staking_tokens": [
+      {
+        "denom": "udaric"
+      }
+    ]
+  },
+  "codebase": {
+    "git_repo": "https://github.com/desmos-labs/desmos",
+    "recommended_version": "v5.0.0",
```

Comment:
based on the versions list, shouldn't this (and the compatible_versions list) be v5.2.0 ?

Reply:
It seems that on the testnet, the [upgrade proposal to v5.2.0](https://testnet.bigdipper.live/desmos/it/proposals/47) did not pass: https://testnet.bigdipper.live/desmos/it/proposals.
What do you suggest to do in this scenario?

Reply:
In this case I'd suggest removing v5.2.0 object from the versions array, and this 'recommended version should be whatever the current version is (v5.1.0?)

Reply:
Okay, that sounds very reasonable. I'll provide the correct implementation 🚀

Reply:
I provided the modification from your suggestion following the previous version of the desmos (not demostestnet) configuration file (including the v5.2.0) [here](https://github.com/cosmos/chain-registry/blob/b26c994570f11209b8f6f1155d53e3610fc2f47a/desmos/chain.json).
Do you think it is okay now?